### PR TITLE
Created AU directory and AU model

### PR DESCRIPTION
**Pull request by ssalsi**
The model was created based on the [research colab](https://colab.corp.google.com/drive/1hzpuvIf7rigNa4DJLfovcqk8a51sXFsX?resourcekey=0-VlQ_SAKqfVaPWK8G6qY6sQ#scrollTo=xe_fbvPB24OO) and the [doc conclusions](https://docs.google.com/document/d/17Cra84Rdkrbx-JB88vpO7uYJ4E1WlyJRlieuhPwwvuo/edit?tab=t.0#heading=h.kh0bgxn0s8oe).

Also, additionally, amended .gitignore, since I'm running the repoo on cloudtop and in order for packages to be installed we need to use Virtual Environment (.venv).

### CODE REVIEW

**Review by norgevz — COMMENTED** [review]
Could you run `python3 main.py` and include the generated HTML content in the PR?

File: model/countries/AU/AU-model.yaml
Comment (norgevz) [inline model/countries/AU/AU-model.yaml:17]: `street-name` and `house-number` have no directly mapping today in Chrome. We only generate field types up to the `street` and `building` levels.  I would recommend to keep the entire `building-location` node as it would capture both concept. 

Also, reading through [1] (it was quite informative) I understood that we could also find Floors in the address, so we should probably capture that as well 

```
street-address-alternative-1:
- building-location
- in-building-location
    - floor
    - unit
```
That means we should probably add `in-building-location` to the `cut-off-children` list.

[1]https://auspost.com.au/content/dam/auspost_corp/media/documents/australia-post-addressing-standards-1999.pdf

Comment (norgevz) [inline model/countries/AU/AU-model.yaml:17]: street-name and house number have no directly mapping today in Chrome. We only generate field types up to the `street` and `building`.  I would recommend to keep the entire `building-location` node as it would capture both concept. 

Also, reading through [1] (it was quite informative) I understood that we could also find Floors in the address, so we should probably capture that as well 

```
street-address-alternative-1:
- building-location
- in-building-location
    - floor
    - unit
```
That means we should probably add `in-building-location` to the `cut-off-children` list.

Comment (ssalsi) [reply model/countries/AU/AU-model.yaml]: Adjusted the changes, as we discussed!
Thanks!

File: model/countries/AU/AU-model.yaml
Comment (norgevz) [inline model/countries/AU/AU-model.yaml:15]: I think you can drop these values. `cross-streets` won't be already present in the model due to the redefinition of the address alternative. `building-name` and `street-type` will just not be generated by our transpiler.

**Review by norgevz — APPROVED** [review]
LGTM % one ask. 

Please wait for Dominic's review before landing.

Thank you!

File: model/countries/AU/AU-model.yaml
Comment (battre) [reply model/countries/AU/AU-model.yaml:15]: I would recommend to follow the US style and have

```
cut-off-children:
  - street
  - family-name
```

instead of listing `street-type` here. Or do you have a specific reason why you added it?

File: model/countries/AU/AU-model.yaml
Comment (battre) [inline model/countries/AU/AU-model.yaml:14]: This is still listed in AU.html. Did you regenerate it?

I would recommend adding `- building` to `cut-off-children:`

Comment (ssalsi) [reply model/countries/AU/AU-model.yaml]: Added both `-building` and `-street` to cut-offs and regenerated AU.html. PTAL!

File: model/countries/AU/AU-model.yaml
Comment (ssalsi) [reply model/countries/AU/AU-model.yaml]: Adjusted the change!

**Pull request merged**
